benchmarks/serialization_libraries/support/serializers/user_serializer.rb

```diff
@@ -5,3 +5,64 @@ class UserSerializer < ActiveModel::Serializer
 
   has_many :posts#, each_serializer: PostSerializer
 end
+
+require 'ams/document_serializer'
+module AMS
```

Comment:
this is _very_ manual.

Reply:
Implantation was incomplete so I pieced it together here :)

Reply:
Not sure I understand the purpose of the comparison with `ams_dev`, as in its current form it is not comparable with the other solutions in terms of functionalities.
It would make more sense to compare against a pre-built document hash, and just timing the `.to_json` call.

Reply:
Was just exploring locally and pushed to share what I did and saw. I've now removed the ams-dev commits and force pushed

Reply:
Right, I see. It would still be interesting to bench against a baseline that would be simply turning a hash into json, since it is a significant part of the time spent.